web/src/state/utils/atomDefault.ts

```diff
@@ -0,0 +1,41 @@
+import type { AtomEffect, GetCallback, GetRecoilValue, RecoilState, SerializableParam } from 'recoil'
+import { atom, atomFamily, selector, selectorFamily } from 'recoil'
+
+export interface AtomDefaultOptions<T> {
+  key: string
+  effects?: ReadonlyArray<AtomEffect<T>>
+  effects_UNSTABLE?: ReadonlyArray<AtomEffect<T>>
+  dangerouslyAllowMutability?: boolean
+  default: (opts: { get: GetRecoilValue; getCallback: GetCallback }) => T
+}
```

Comment:
This is essentially the type of the argument of `atom` except that we modify `default`, isn't it? We could also express that as a type more directly:
```suggestion
export type AtomDefaultOptions<T> = Omit<AtomOptions<T>, 'default'> & {
  default: (opts: { get: GetRecoilValue; getCallback: GetCallback }) => T
}
```

Reply:
Nice suggestion!